resources/authgear/templates/en/translation.json

```diff
@@ -88,6 +88,7 @@
   "copy-button-label": "Copy",
   "copied-button-label": "Copied",
   "get-new-code-button-label": "Get new code",
```

Comment:
Seems it is replaced by `get-new-recovery-codes-button-label` and it is not used. Remove it?